RELEASE/scripts/autoscend/paths/small.ash

```diff
@@ -17,7 +17,17 @@ void small_initializeSettings()
 	{
 		//having vastly lower stats and no easy solutions in hardcore means you always die from flyering
 		//should be replaced with a more elegant solution where detailed estimation / calculation is done.
-		set_property("auto_ignoreFlyer", true);
+		//set_property("auto_ignoreFlyer", true);
+
+		//cap ML to 50 to help avoid getting beaten up
+		int MLCap = 50;
+		string MLSafetyLimit = get_property("auto_MLSafetyLimit");
+		if (MLSafetyLimit == "" || MLSafetyLimit.to_int() > MLCap)
+		{
+			// record existing MLSafetyLimit so it can be restored at end of run
+			set_property("auto_MLSafetyLimitBackup",MLSafetyLimit);
+			set_property("auto_MLSafetyLimit",MLCap);
```

Comment:
After seeing the issues I had with ML, do we want to also add a backup for auto_disregardInstantKarma and set it to false or just leave users on their own?

Reply:
Didn't hear. Did making auto_disregardInstantKarma fix getting beaten up for ya?